tests/patient.test.js

```diff
@@ -88,4 +93,45 @@ describe('Patient', function () {
       })
     })
   })
+
```

Comment:
@Hitenp1 moved your patient route tests into here to consolidate all patient tests